pkg/vdri/trustbloc/vdri.go

```diff
@@ -21,29 +23,44 @@ import (
 
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/config/httpconfig"
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/config/memorycacheconfig"
+	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/config/signatureconfig"
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/config/verifyingconfig"
+	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/didconfiguration"
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/discovery/staticdiscovery"
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/endpoint"
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/models"
 	"github.com/trustbloc/trustbloc-did-method/pkg/vdri/trustbloc/selection/staticselection"
 )
 
+type configService interface {
+	GetConsortium(string, string) (*models.ConsortiumFileData, error)
+	GetStakeholder(string, string) (*models.StakeholderFileData, error)
+}
+
 type endpointService interface {
 	GetEndpoints(domain string) ([]*models.Endpoint, error)
 }
 
+type didConfigService interface {
+	VerifyStakeholder(domain string, doc *docdid.Doc) error
+}
+
 type vdri interface {
 	Build(pubKey *vdriapi.PubKey, opts ...vdriapi.DocOpts) (*docdid.Doc, error)
 	Read(did string, opts ...vdriapi.ResolveOpts) (*docdid.Doc, error)
 }
 
 // VDRI bloc
 type VDRI struct {
-	resolverURL     string
-	endpointService endpointService
-	getHTTPVDRI     func(url string) (vdri, error) // needed for unit test
-	tlsConfig       *tls.Config
-	authToken       string
+	resolverURL      string
+	configService    configService
+	endpointService  endpointService
+	didConfigService didConfigService
+	getHTTPVDRI      func(url string) (vdri, error) // needed for unit test
```

Comment:
I know this is not part of your change so you don't have to fix it, but this is unusual